ios/FluentUI/Core/Colors.swift

```diff
@@ -311,21 +310,21 @@ public final class Colors: NSObject {
     }
 
     public struct BarButtonItem {
-        public static var primary: UIColor = foreground8
+        public static var primary: UIColor { return UIColor(light: Colors.primary, dark: .white) }
         public static var secondary: UIColor = foreground2d
```

Comment:
Out of curiosity, so in this change we're making the primary color settable by the clients live, but not the secondary? Is that something we'd want to do in the future or not something we expect to be a client pain point?

NOT SAYING TO HANDLE DOCUMENTATION OF THIS CLASS IN THIS CHANGE: this is another case where "simple" or "trivial" documentation of APIs would allow us to adapt it to indicate that some of these properties are readonly-computed and others are just value assignments. A client could before this change call 
```Swift
Colors.BarButtonItem.primary = foo
Colors.BarButtonItem.secondary = bar
```

But now the first line will give an error while the second line would not. Maybe this is a desired outcome because it forces consistency with the primary colors and gives freedom to clients with secondary colors, but it's hard to understand just by looking at the code without any documentation.

Reply:
Yeah it seems hard to find the write documentation right now. what if I write, this comment in the mean time 
`    /// Variation of App brand colors. If an application is a hub of different apps, `primary` color could change within the same foreground session.
    /// It is recomended not to cache `primary` color because it could change.
 
@objc public static var primary: UIColor = communicationBlue`

Reply:
Sounds good!